odxtools/isotp_state_machine.py

```diff
@@ -128,7 +131,7 @@ async def read_telegrams(self, bus: Union[can.Bus,
                 for tmp in self.decode_rx_frame(msg.arbitration_id, msg.data):
                     yield tmp
         else:
-            assert isinstance(bus, TextIO)
+            assert isinstance(bus, (TextIO, TextIOWrapper))
```

Comment:
@andlaus from I have seen, TextIO is not runtime checkable, or at least there is nothing in the official docs to indicate it is

If you want to do runtime check, you most likely need to check on `io.TextIOBase`

Reply:
Thanks for your input. I added `io.TextIOBase` instead of `typing.TextIO` in the last commit.